applications/StructuralMechanicsApplication/custom_response_functions/structural_response.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import print_function, absolute_import, division
+
+# importing the Kratos Library
+from KratosMultiphysics import *
+import structural_mechanics_analysis
+
+import time as timer
+
+class ResponseFunctionBase(object):
+    def __init__(self, identifier, project_parameters):
+        self.identifier = identifier
+        self.project_parameters = project_parameters
+    def Initialize(self):
+        pass
+    def CalculateValue(self):
```

Comment:
Is it just me or wouldn't it be nicer to have blank lines between the function definitions?